### PR TITLE
Improve breadcrumbs and version verbiage

### PR DESCRIPTION
**Pull request by tsibley**
See commit messages for details.

### CODE REVIEW

**Comment by tsibley** [comment]
Breadcrumbs before/after on main docs site:

image
image

on Augur's docs:

image
image

(ignore the "View page source" vs. "Edit on GitHub" difference, which is not real.)

File: lib/nextstrain/sphinx/theme/layout.html [hunk 1]
```diff
@@ -49,17 +45,19 @@
       {%- set nav_version = current_version %}
     {% endif %}
     {% if nav_version %}
-      <span>
-        version: {{ nav_version }}
-      </span>
+      <div class="version">
+        {% if nav_version == "stable" %}
+          {# don't show anything #}
```
Comment (huddlej) [inline lib/nextstrain/sphinx/theme/layout.html:50]: Could we report version here, too, along with  "stable" to fix [this related issue](https://github.com/nextstrain/augur/issues/817)?

File: lib/nextstrain/sphinx/theme/theme.conf [hunk 2]
```diff
@@ -8,3 +8,4 @@ pygments_style = default
 style_nav_header_background = #f2f2f2
 logo = True
 logo_link = https://docs.nextstrain.org
+subproject = True
```
Comment (huddlej) [inline lib/nextstrain/sphinx/theme/theme.conf:11]: Do each of these config attributes appear in the template namespace as `theme_{param}`? Just making sure I understand how this maps to `theme_subproject` in the template code.

File: lib/nextstrain/sphinx/theme/breadcrumbs.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  {% for doc in parents %}
+    <li><a href="{{ doc.link|e }}">{{ doc.title }}</a></li>
+  {% endfor %}
+
+  {#
+    Don't include the current page's title (like the template we're extending
+    does) because the page title is immediately below these breadcrumbs anyway.
+
+    This also follows common UX recommendations for breadcrumbs, e.g. the UK's
+    Government Digital Service (gov.uk) breadcrumbs component guidelines¹ says:
```
Comment (huddlej) [inline lib/nextstrain/sphinx/theme/breadcrumbs.html:34]: Boris approves this message. ;)

**Comment by tsibley** [comment]
Version verbiage before/after for version X:

image
image

for version "latest":

image
image

for version "stable":

image
image

**Review by huddlej — APPROVED** [review]
Beautiful. Thank you! I had a question about a related issue that I've mentioned inline. It's ok if that's off-topic for this PR though.

File: lib/nextstrain/sphinx/theme/theme.conf (hunk 2 above)
Comment (tsibley) [reply lib/nextstrain/sphinx/theme/theme.conf:11]: Yep, they walk right in just like that and make themselves at home in the global template context.

File: lib/nextstrain/sphinx/theme/layout.html (hunk 1 above)
Comment (tsibley) [reply lib/nextstrain/sphinx/theme/layout.html:50]: _Maybe_? If `version` still retains the actual value when `nav_version` is `latest`, then yes. Reading the [RTD issue](https://github.com/readthedocs/readthedocs.org/issues/4529) linked from our Augur issue, it seems like `version` will retain its value. I can try things out, but will do it on another PR.

Comment (tsibley) [reply lib/nextstrain/sphinx/theme/layout.html:50]: Trialing things over in https://github.com/nextstrain/sphinx-theme/pull/16

**Pull request merged**